packages/components/inputs/select-input/src/select-input.tsx

```diff
@@ -161,7 +161,7 @@ export type TSelectInputProps = {
   // formatOptionLabel: PropTypes.func,
   // getOptionLabel: PropTypes.func,
   // getOptionValue: PropTypes.func,
-  // hideSelectedOptions: PropTypes.bool,
+  hideSelectedOptions?: ReactSelectProps['hideSelectedOptions'];
```

Comment:
Inputs are a separate set of work, but there are some basic props, like `hideSelectedOptions`, that we should make available for every `react-select`-based input